xbmc/input/linux/LinuxInputDevices.cpp

```diff
@@ -280,6 +280,14 @@ KeyMap keyMap[] = {
   { 378               , XBMCK_RIGHT       }, // Green
   { 381               , XBMCK_UP          }, // Yellow
   { 366               , XBMCK_DOWN        }, // Blue
+  // Wiimote controller Additions
+  { 304               , XBMCK_RETURN      }, // A
+  { 305               , XBMCK_ESCAPE      }, // B
```

Comment:
we have to stop those hacks. symbols are defined in input-event-codes. 305 is already defined: https://github.com/torvalds/linux/blob/master/include/uapi/linux/input-event-codes.h#L382